components/library/src/slugs.rs

```diff
@@ -1,7 +1,7 @@
 pub use slug::slugify;
 
 pub fn quasi_slugify(s: &str) -> String {
-    s.replace("#", "").replace("/", "")
+    s.replace("#", "").replace("/", "").replace("?", "").replace("\n", "")
```

Comment:
@southerntofu something I just remembered if that we should also removed invalid Windows characters in filenames like `:`.
It is already an issue in the page filename (https://www.getzola.org/documentation/content/page/): 

> The page name will be anything after _/- so a filename like 2018-10-10-hello-world.md will be available at [base_url]/hello-world. Note that the full RFC3339 datetime contains colons, which is not a valid character in a filename on Windows.

The full list: https://gist.github.com/doctaphred/d01d05291546186941e1b7ddc02034d3#file-ntfs-filenames-txt-L9-L17
Also needs to `trim_end` apparently: https://gist.github.com/doctaphred/d01d05291546186941e1b7ddc02034d3#file-ntfs-filenames-txt-L33

The full RFC3339 was already a mistake imo so let's not repeat it for actual pages/sections and be fully compatible with every OS.

Reply:
I think this is solved now. Also i updated the docs.